staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external.go

```diff
@@ -46,7 +46,7 @@ import (
 // each is needed.
 func (g *Cloud) ensureExternalLoadBalancer(clusterName string, clusterID string, apiService *v1.Service, existingFwdRule *compute.ForwardingRule, nodes []*v1.Node) (*v1.LoadBalancerStatus, error) {
 	if len(nodes) == 0 {
-		return nil, fmt.Errorf("Cannot EnsureLoadBalancer() with no hosts")
+		return nil, fmt.Errorf("cannot EnsureLoadBalancer() with no hosts")
```

Comment:
To fix tests, I think it would be better to replace the error string here with `errStrLbNoHosts` and fix the capitalization there:

https://github.com/kubernetes/kubernetes/blob/28366a1db401d2c0e1652870a68e690f96a1e9cf/staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_utils_test.go#L46

Reply:
@nikhita Yes, now using shared variables here 7c73a3f

Reply:
Test `pull-kubernetes-integration` keeps failing 😭